pkg/helm/watches/watches.go

```diff
@@ -176,7 +175,7 @@ func UpdateForResource(path string, r *scaffold.Resource, chartName string) (err
 	}
 	watch := Watch{
 		GroupVersionKind: gvk,
-		ChartDir:         filepath.Join(helm.HelmChartsDir, chartName),
+		ChartDir:         filepath.Join("helm-charts", chartName),
```

Comment:
Why are we not using the constant here? [HelmChartsDir](https://github.com/operator-framework/operator-sdk/blob/6e62407c90193cfc531c017a38031a7d82691f3e/internal/scaffold/helm/chart.go#L41) is still in use in elsewhere in this PR: 
- https://github.com/operator-framework/operator-sdk/pull/3206/files#diff-12be4260f4a1ac30453240719db440a4R41
- https://github.com/operator-framework/operator-sdk/pull/3206/files#diff-b40429d0611057328d1e0a98add0b6acR41

Reply:
If we use the const from https://github.com/operator-framework/operator-sdk/blob/6e62407c90193cfc531c017a38031a7d82691f3e/internal/scaffold/helm/chart.go#L41 we will face ` import cycle not allowed`. So, in order to keep it here we cannot use it. 

However, I am not too worry because I understand that we will refac these implementations very soon